blosc/blosc-comp-features.h

```diff
@@ -0,0 +1,23 @@
+/*********************************************************************
+  Blosc - Blocked Shuffling and Compression Library
+
+  Author: Francesc Alted <francesc@blosc.org>
+
+  See LICENSES/BLOSC.txt for details about copyright and rights to use.
+**********************************************************************/
+
+#ifndef BLOSC_COMP_FEATURES_H
+#define BLOSC_COMP_FEATURES_H
+
+/* Use inlined functions for supported systems */
+#if defined(_MSC_VER) && !defined(__cplusplus)   /* Visual Studio */
+  #define BLOSC_INLINE __inline  /* Visual C is not C99, but supports some kind of inline */
+#else
+  #if defined __STDC__ && defined __STDC_VERSION__ && __STDC_VERSION__ >= 199901L

```

Comment:
Which could be written shorter as
```c
#elif __STDC_VERSION__ >= 199901L
```
Remember undefined macros evaluate to 0.

Reply:
Good point.  I'll fix this after merging.